mne/annotations.py

```diff
@@ -706,12 +708,12 @@ def _write_annotations(fid, annotations):
     end_block(fid, FIFF.FIFFB_MNE_ANNOTATIONS)
 
 
-def _write_annotations_csv(fname, annot):
+def _write_annotations_csv(fname, annot, sep=','):
     annot = annot.to_data_frame()
     if 'ch_names' in annot:
         annot['ch_names'] = [
             _prep_name_list(ch, 'write') for ch in annot['ch_names']]
-    annot.to_csv(fname)
+    annot.to_csv(fname, sep=sep)
```

Comment:
Unrelated to this PR, but shouldn't we pass `index=False` here?

Reply:
Yes!